### PR TITLE
Update uk-google.md

### PR DESCRIPTION
**Pull request by maxulysse**

### CODE REVIEW

**Review by christopher-hakkaart — APPROVED** [review]
It looks to me like your sync is at 5 pm CEST? Was this on purpose? The online sync is 4 pm.

**Comment by maxulysse** [comment]
> It looks to me like your sync is at 5 pm CEST? Was this on purpose? The online sync is 4 pm.

Not at all, I copy pasted from another site that I think war in UK just to avoid this kind of issue

File: markdown/events/2023/hackathon-march-2023/uk-google.md [hunk 1]
```diff
@@ -32,33 +32,36 @@ Please use the [main registration form](https://nf-co.re/events/2023/hackathon-m
             </thead>
             <tbody>
             <tr>
-                <td data-timestamp="1679905800" data-timeformat="HH:mm z">09:30</td>
+                <td data-timestamp="1679905800" data-timeformat="HH:mm z">08:30</td>
```
Comment (mashehu) [inline markdown/events/2023/hackathon-march-2023/uk-google.md:35]:
```suggestion
                <td data-timestamp="1679904000" data-timeformat="HH:mm z">08:30</td>
```
You need to update the timestamps. the other string is overwritten based on the timestamp (to always show local time)

Comment (maxulysse) [reply markdown/events/2023/hackathon-march-2023/uk-google.md:35]: Not sure with GMT->BST, but I think I had the right time

File: markdown/events/2023/hackathon-march-2023/uk-google.md [hunk 2]
```diff
@@ -3,17 +3,71 @@ title: Hackathon - March 2023 (Google Academy)
...
 location_name: Google Academy, London
 address: 123 Buckingham Palace Rd, London SW1W 9SH
 location_url: https://www.wired.co.uk/article/google-digital-skills-academy
 location_latlng: [51.493721, -0.146830]
 ---
 
-Local event to be held at the Google Academy, London.
+Local event to be held at the [Google Academy, London](https://goo.gl/maps/4RH3qsYMRVnM65Bb9).
 
 Primary contact: [<i class="fab fa-slack"></i> Harshil Patel](https://nfcore.slack.com/team/UEB97FBN3)
```
Comment (drpatelh) [inline markdown/events/2023/hackathon-march-2023/uk-google.md:17]:
```suggestion
Primary contact(s): [<i class="fab fa-slack"></i> Harshil Patel](https://nfcore.slack.com/team/UEB97FBN3)
```

File: markdown/events/2023/hackathon-march-2023/uk-google.md [hunk 3]
```diff
@@ -3,17 +3,71 @@ title: Hackathon - March 2023 (Google Academy)
...
+            </tr>
+            </thead>
+            <tbody>
+            <tr>
+                <td data-timestamp="1679905800" data-timeformat="HH:mm z">09:30</td>
+                <td background-color:navy; rowspan="1">Welcome and Setup</td>
+                <td background-color:navy; rowspan="1">Daily Start</td>
+                <td background-color:navy; rowspan="1">Daily Start</td>
+            </tr>
+                <td data-timestamp="1679907600" data-timeformat="HH:mm z">09:45</td>
```
Comment (drpatelh) [inline markdown/events/2023/hackathon-march-2023/uk-google.md:40]:
```suggestion
                <td data-timestamp="1679907600" data-timeformat="HH:mm z">10:00</td>
```

File: markdown/events/2023/hackathon-march-2023/uk-google.md [hunk 4]
```diff
@@ -3,17 +3,71 @@ title: Hackathon - March 2023 (Google Academy)
...
+                <td rowspan="1">Intro by hub organiser</td>
+                <td rowspan="2">Hack!</td>
+                <td rowspan="2">Hack!</td>
+            </tr>
+            <tr>
+                <td data-timestamp="1679911200" data-timeformat="HH:mm z">10:00</td>
```
Comment (drpatelh) [inline markdown/events/2023/hackathon-march-2023/uk-google.md:46]:
```suggestion
                <td data-timestamp="1679911200" data-timeformat="HH:mm z">10:30</td>
```

File: markdown/events/2023/hackathon-march-2023/uk-google.md [hunk 5]
```diff
@@ -3,17 +3,71 @@ title: Hackathon - March 2023 (Google Academy)
...
+                <th>Time</th>
+                <th>Mon. 27 Mar., 2023</th>
+                <th>Tue. 28 Mar., 2023</th>
+                <th>Wed. 29 Mar., 2023</th>
+            </tr>
+            </thead>
+            <tbody>
+            <tr>
+                <td data-timestamp="1679905800" data-timeformat="HH:mm z">09:30</td>
+                <td background-color:navy; rowspan="1">Welcome and Setup</td>
```
Comment (drpatelh) [inline markdown/events/2023/hackathon-march-2023/uk-google.md:36]:
```suggestion
                <td background-color:navy; rowspan="1">Registations open</td>
```

File: markdown/events/2023/hackathon-march-2023/uk-google.md [hunk 6]
```diff
@@ -3,17 +3,78 @@ title: Hackathon - March 2023 (Google Academy)
...
```
Comment (drpatelh) [inline markdown/events/2023/hackathon-march-2023/uk-google.md:35]:
```suggestion
                <td background-color:navy; rowspan="1">Registrations open</td>
```

File: markdown/events/2023/hackathon-march-2023/uk-google.md [hunk 7]
```diff
@@ -3,17 +3,78 @@ title: Hackathon - March 2023 (Google Academy)
...
+                <td background-color:navy; rowspan="1">Daily Start</td>
+                <td background-color:navy; rowspan="1">Daily Start</td>
+            </tr>
+                <td data-timestamp="1679907600" data-timeformat="HH:mm z">10:00</td>
+                <td rowspan="1">Intro by nf-core</td>
```
Comment (drpatelh) [inline markdown/events/2023/hackathon-march-2023/uk-google.md:40]:
```suggestion
                <td rowspan="1">Welcome by Harshil Patel</td>
```

File: markdown/events/2023/hackathon-march-2023/uk-google.md [hunk 8]
```diff
@@ -3,17 +3,78 @@ title: Hackathon - March 2023 (Google Academy)
...
+                <td rowspan="4">Hack!</td>
+                <td rowspan="4">Hack!</td>
+            </tr>
+            <tr>
+                <td data-timestamp="1679911200" data-timeformat="HH:mm z">10:15</td>
+                <td rowspan="1">Intro by Google</td>
```
Comment (drpatelh) [inline markdown/events/2023/hackathon-march-2023/uk-google.md:47]:
```suggestion
                <td rowspan="1">Hackathon introduction by Maxime Garcia</td>
```

File: markdown/events/2023/hackathon-march-2023/uk-google.md [hunk 9]
```diff
@@ -3,17 +3,78 @@ title: Hackathon - March 2023 (Google Academy)
...
+            </tr>
+            <tr>
+                <td data-timestamp="1679911200" data-timeformat="HH:mm z">10:30</td>
+                <td rowspan="1">Hackathon intro</td>
```
Comment (drpatelh) [inline markdown/events/2023/hackathon-march-2023/uk-google.md:51]:
```suggestion
                <td rowspan="1">Welcome by Google</td>
```

File: markdown/events/2023/hackathon-march-2023/uk-google.md [hunk 10]
```diff
@@ -3,17 +3,79 @@ title: Hackathon - March 2023 (Google Academy)
...
+            <tr>
+                <th>Time</th>
+                <th>Mon. 27 Mar., 2023</th>
+                <th>Tue. 28 Mar., 2023</th>
+                <th>Wed. 29 Mar., 2023</th>
+            </tr>
+            </thead>
+            <tbody>
+            <tr>
+                <td data-timestamp="1679905800" data-timeformat="HH:mm z">09:30</td>
```
Comment (maxulysse) [inline markdown/events/2023/hackathon-march-2023/uk-google.md:35]:
```suggestion
                <td data-timestamp="1679913000" data-timeformat="HH:mm z">09:30</td>
```

File: markdown/events/2023/hackathon-march-2023/uk-google.md [hunk 11]
```diff
@@ -3,17 +3,79 @@ title: Hackathon - March 2023 (Google Academy)
...
+                <td background-color:navy; rowspan="1">Registrations open</td>
+                <td background-color:navy; rowspan="1">Daily Start</td>
+                <td background-color:navy; rowspan="1">Daily Start</td>
+            </tr>
+                <td data-timestamp="1679907600" data-timeformat="HH:mm z">10:00</td>
```
Comment (maxulysse) [inline markdown/events/2023/hackathon-march-2023/uk-google.md:40]:
```suggestion
                <td data-timestamp="1679914800" data-timeformat="HH:mm z">10:00</td>
```

File: markdown/events/2023/hackathon-march-2023/uk-google.md [hunk 12]
```diff
@@ -3,17 +3,79 @@ title: Hackathon - March 2023 (Google Academy)
...
+                <td rowspan="1">Welcome by nf-core</td>
+                <td rowspan="4">Hack!</td>
+                <td rowspan="4">Hack!</td>
+            </tr>
+            <tr>
+                <td data-timestamp="1679911200" data-timeformat="HH:mm z">10:15</td>
+                <td rowspan="1">Hackathon introduction</td>
+            </tr>
+            <tr>
+                <td data-timestamp="1679911200" data-timeformat="HH:mm z">10:30</td>
```
Comment (maxulysse) [inline markdown/events/2023/hackathon-march-2023/uk-google.md:50]:
```suggestion
                <td data-timestamp="1679916600" data-timeformat="HH:mm z">10:30</td>
```

File: markdown/events/2023/hackathon-march-2023/uk-google.md [hunk 13]
```diff
@@ -3,17 +3,79 @@ title: Hackathon - March 2023 (Google Academy)
...
+                <td rowspan="1">Welcome by Google</td>
+            </tr>
+            <tr>
+                <td data-timestamp="1679911200" data-timeformat="HH:mm z">11:00</td>
```
Comment (maxulysse) [inline markdown/events/2023/hackathon-march-2023/uk-google.md:54]:
```suggestion
                <td data-timestamp="1679875200" data-timeformat="HH:mm z">11:00</td>
```

File: markdown/events/2023/hackathon-march-2023/uk-google.md [hunk 14]
```diff
@@ -3,17 +3,79 @@ title: Hackathon - March 2023 (Google Academy)
...
+                <td rowspan="1">Hack!</td>
+            </tr>
+            <tr>
+                <td data-timestamp="1679918400" data-timeformat="HH:mm z">13:00</td>
```
Comment (maxulysse) [inline markdown/events/2023/hackathon-march-2023/uk-google.md:58]:
```suggestion
                <td data-timestamp="1679925600" data-timeformat="HH:mm z">13:00</td>
```

File: markdown/events/2023/hackathon-march-2023/uk-google.md [hunk 15]
```diff
@@ -3,17 +3,79 @@ title: Hackathon - March 2023 (Google Academy)
...
+                <td rowspan="1">Lunch</td>
+            </tr>
+            <tr>
+                <td data-timestamp="1679922000" data-timeformat="HH:mm z">14:00</td>
+                <td rowspan="2">Hack!</td>
+                <td rowspan="2">Hack!</td>
+                <td rowspan="1">Hack!</td>
+            </tr>
+            <tr>
+                <td data-timestamp="1679929200"  data-timeformat="HH:mm z">15:30</td>
```
Comment (maxulysse) [inline markdown/events/2023/hackathon-march-2023/uk-google.md:70]:
```suggestion
                <td data-timestamp="1679934600"  data-timeformat="HH:mm z">15:30</td>
```

File: markdown/events/2023/hackathon-march-2023/uk-google.md [hunk 16]
```diff
@@ -3,17 +3,79 @@ title: Hackathon - March 2023 (Google Academy)
...
+                <td>Final wrap-up</td>
+            </tr>
+            <tr>
+                <td data-timestamp="1679925600"  data-timeformat="HH:mm z">17:00</td>
```
Comment (maxulysse) [inline markdown/events/2023/hackathon-march-2023/uk-google.md:74]:
```suggestion
                <td data-timestamp="1679940000"  data-timeformat="HH:mm z">17:00</td>
```

**Pull request merged**